tests/integration/Owncloud/OcisPhpSdk/DriveTest.php

```diff
@@ -170,4 +170,52 @@ public function testCreateDriveInvite(): void
             $this->fail("EndPointNotImplementedException was thrown unexpectedly");
         }
     }
+
+    /**
+     * @throws ForbiddenException
+     * @throws InvalidResponseException
+     * @throws BadRequestException
+     * @throws EndPointNotImplementedException
+     * @throws UnauthorizedException
+     * @throws HttpException
+     * @throws NotFoundException
+     * @throws InternalServerErrorException
+     * @throws \Exception
+     */
+    public function testDeleteDriveInvite(): void
+    {
+        $this->initUser('marie', 'radioactivity');
+
+        $marie = $this->ocis->getUsers('marie')[0];
+
+        $managerRole = null;
+
+        if (getenv('OCIS_VERSION') === "stable") {
+            $this->markTestSkipped('Ocis version < 6.0.0 does not support creation of drive share invite so delete drive share test has been skipped');
```

Comment:
This is correct. It is difficult, or "impossible", to test what happens in ocis V5 if someone tries to delete a drive share, because it is not possible to create a drive share in the first place.

Someone could try to delete a "bogus" drive share, putting some non-existent id. But that would be the same as any "completely wrong" attempt to use the non-existent end-point, and that is already covered by ocis tests.